src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -67,8 +67,13 @@ final class ProgressBar
     /**
      * @param int $max Maximum steps (0 if unknown)
      */
-    public function __construct(OutputInterface $output, int $max = 0, float $minSecondsBetweenRedraws = 1 / 25)
-    {
+    public function __construct(
+        OutputInterface $output, 
```

Comment:
Please keep the signature on a single line. We wrap signatures only when using promoted properties